app/mailers/producer_mailer.rb

```diff
@@ -12,12 +12,14 @@ def order_cycle_report(producer, order_cycle)
 
     subject = "[#{Spree::Config.site_name}] #{I18n.t('producer_mailer.order_cycle.subject', producer: producer.name)}"
 
-    if has_orders? order_cycle, producer
-      mail(to: @producer.email,
-           from: from_address,
-           subject: subject,
-           reply_to: @coordinator.email,
-           cc: @coordinator.email)
+    if has_orders?(order_cycle, producer)
```

Comment:
I think it's a bit too late to check whether the order cycle has orders. It's pointless to send a notification if there are none. IMO we should check before enqueuing the `OrderCycleNotificationJob`. I'll create an issue for it.

Reply:
done